dev/src/components/ReviewPage.tsx

```diff
@@ -7,18 +7,24 @@ import ReviewForm from "./manual-grading/ReviewForm"
 import DynamicTable, { RowElement } from "./DynamicTable"
 import Button, { ButtonType } from "./admin/Button"
 import Release from "./Release"
+import { useHistory } from "react-router"
 
 
 const ReviewPage = (): JSX.Element => {
     const state = useAppState()
     const actions = useActions()
     const courseID = getCourseID()
-    const isCourseCreator = state.courses[courseID].getCoursecreatorid() === state.self.getId()
+    const history = useHistory()
     const [groupView, setGroupView] = useState<boolean>(false)
 
     useEffect(() => {
         if (!state.courseSubmissions[courseID]) {
-            actions.getAllCourseSubmissions(courseID)
+            // Redirect to home if the call to get all submissions fails
```

Comment:
Will that help? Does it mean the user must login again?

Reply:
After changing the behaviour of `success(response)` to reload the page if the user session is expired/invalid, this is no longer necessary.